flink-quickstart/flink-quickstart-scala/src/main/resources/archetype-resources/pom.xml

```diff
@@ -45,6 +45,7 @@ under the License.
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<flink.version>@project.version@</flink.version>
+		<target.java.version>@target.java.version@</target.java.version>
```

Comment:
Maybe also update this field in the `flink-quickstart-java` module. I think there it is fixed to `1.8`.

Reply:
Shouldn't we also update `maven-compiler-plugin` in this `pom.xml` to use `${target.java.version}`?

Reply:
Those are good suggestions and will help us down the line once we default to Java 11.

Currently it still should've worked though; because the goal was to just get the scala compiler to work with java 8 instead of 6.